README.md

```diff
@@ -202,17 +202,21 @@ Autometrics makes it easy to identify if a specific version or commit introduced
 >
 > autometrics-py will track support for build_info using the OpenTelemetry tracker via [this issue](https://github.com/autometrics-dev/autometrics-py/issues/38)
 
-The library uses a separate metric (`build_info`) to track the version and, optionally, the git commit of your service.
+The library uses a separate metric (`build_info`) to track the version and git metadata of your code - repository url, provider name, commit and branch.
 
-It then writes queries that group metrics by the `version`, `commit` and `branch` labels so you can spot correlations between code changes and potential issues.
+It then writes queries that group metrics by the set labels so you can spot correlations between code changes and potential issues.
```

Comment:
reword: "It then writes queries that group metrics by these labels, so you can spot correlations between code changes and potential issues."

Reply:
holdup, i forgot i can suggest a change directly in github, gonna do that